ee/modules/600-flant-integration/images/madison-proxy/rootfs/etc/nginx/madison.tmpl

```diff
@@ -25,6 +25,8 @@ server {
 
     rewrite /api/v2/alerts /api/events/prometheus/${MADISON_AUTH_KEY} break;
 
+    rewrite /api/v1/alerts /api/events/prometheus/${MADISON_AUTH_KEY} break;
```

Comment:
Could you leave a comment here about why we need to support both v1 and v2 endpoints for madison proxy?

Reply:
i've added